subiquity/models/tests/test_filesystem.py

```diff
@@ -78,3 +159,702 @@ def test_errors(self):
                     self.fail(
                         "dehumanize_size({!r}) did not error".format(input))
                 self.assertEqual(expected_error, actual_error)
+
+
+MB = 1 << 30
+GB = 1 << 40
+
+
+class TestFilesystemDisk(CiTestCase):
+    def setUp(self):
+        super(TestFilesystemDisk, self).setUp()
+        self.info = argparse.Namespace()
+        self.info.serial = 'myserial'
```

Comment:
In general I prefer to have a test factory that makes strings whose content is immaterial to the test, so stuff like

```
disk.serial = self.factory.makeString()
disk.path = self.factory.makeString()
self.assertEqual(disk.label, disk.serial)
```

rather than hardcoding strings where it can be tricky for the reader to know or not if the contents are important. 

I also dislike setUp() methods in general because of http://xunitpatterns.com/Obscure%20Test.html#Mystery%20Guest sort of things, helper methods to make objects of various kinds are much better IMO.

Reply:
I like helper methods as well;  For complex mocking or setup, I think the helper methods are useful, for simple Object creation, using setUp like we would an __init__() saves function overhead and removes repetition in the unittest itself.

Reply:
Well I disagree but not enough to demand changes in this branch...